{"uid":"5bf5b921fbb22966","name":"新增重复工艺路线","fullName":"TestCase.functional_tests.test_factory_modeling.test_process_related.TestProcessRelated#test_storeProcessRoutingData02","historyId":"fa7b9a357de0a8456bef003321864ccf","time":{"start":1751353611143,"stop":1751353619713,"duration":8570},"description":"新增重复工艺路线，新增失败","descriptionHtml":"<p>新增重复工艺路线，新增失败</p>\n","status":"broken","statusMessage":"TypeError: ProcessRelated.storeProcessRoutingData() got an unexpected keyword argument 'ProcessRoutingCode'","statusTrace":"self = <TestCase.functional_tests.test_factory_modeling.test_process_related.TestProcessRelated object at 0x0000019CF64677D0>\n\n    @pytest.mark.run(order=5)\n    @grade_2\n    @allure.title(\"新增重复工艺路线\")\n    @allure.description(\"新增重复工艺路线，新增失败\")\n    def test_storeProcessRoutingData02(self):\n        with allure.step(\"新增工艺路线\"):\n>           res = ProcessRelated().storeProcessRoutingData(ProcessRoutingCode=self.routing_code)\nE           TypeError: ProcessRelated.storeProcessRoutingData() got an unexpected keyword argument 'ProcessRoutingCode'\n\nTestCase\\functional_tests\\test_factory_modeling\\test_process_related.py:85: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_xunit_setup_class_fixture_TestProcessRelated","time":{"start":1751353611142,"stop":1751353611142,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"attachmentStep":false,"hasContent":false}],"testStage":{"description":"新增重复工艺路线，新增失败","status":"broken","statusMessage":"TypeError: ProcessRelated.storeProcessRoutingData() got an unexpected keyword argument 'ProcessRoutingCode'","statusTrace":"self = <TestCase.functional_tests.test_factory_modeling.test_process_related.TestProcessRelated object at 0x0000019CF64677D0>\n\n    @pytest.mark.run(order=5)\n    @grade_2\n    @allure.title(\"新增重复工艺路线\")\n    @allure.description(\"新增重复工艺路线，新增失败\")\n    def test_storeProcessRoutingData02(self):\n        with allure.step(\"新增工艺路线\"):\n>           res = ProcessRelated().storeProcessRoutingData(ProcessRoutingCode=self.routing_code)\nE           TypeError: ProcessRelated.storeProcessRoutingData() got an unexpected keyword argument 'ProcessRoutingCode'\n\nTestCase\\functional_tests\\test_factory_modeling\\test_process_related.py:85: TypeError","steps":[{"name":"新增工艺路线","time":{"start":1751353611143,"stop":1751353619712,"duration":8569},"status":"broken","statusMessage":"TypeError: ProcessRelated.storeProcessRoutingData() got an unexpected keyword argument 'ProcessRoutingCode'\n","statusTrace":"  File \"D:\\apiAutomationRH\\TestCase\\functional_tests\\test_factory_modeling\\test_process_related.py\", line 85, in test_storeProcessRoutingData02\n    res = ProcessRelated().storeProcessRoutingData(ProcessRoutingCode=self.routing_code)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"attachmentStep":false,"hasContent":true}],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":1,"attachmentStep":false,"hasContent":true},"afterStages":[{"name":"_xunit_setup_class_fixture_TestProcessRelated::0","time":{"start":1751353619722,"stop":1751353619722,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"attachmentStep":false,"hasContent":false}],"labels":[{"name":"feature","value":"工序相关模块"},{"name":"tag","value":"grade_2"},{"name":"tag","value":"run(order=5)"},{"name":"parentSuite","value":"TestCase.functional_tests.test_factory_modeling"},{"name":"suite","value":"test_process_related"},{"name":"subSuite","value":"TestProcessRelated"},{"name":"host","value":"DESKTOP-520CACV"},{"name":"thread","value":"8020-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"TestCase.functional_tests.test_factory_modeling.test_process_related"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[]}],"tags":["run(order=5)","grade_2"]},"source":"5bf5b921fbb22966.json","parameterValues":[]}